core/arch/aarch64/instr_create.h

```diff
@@ -557,6 +572,27 @@
 #define INSTR_CREATE_sub_shimm(dc, rd, rn, rm_or_imm, sht, sha) \
   INSTR_CREATE_sub_shift(dc, rd, rn, rm_or_imm, sht, sha)
 
+
+/**
+ * Creates a FMUL vector instruction.
+ * \param dc The void * dcontext used to allocate memory for the instr_t.
+ * \param Rd output register
+ * \param Rm input register
+ * \param Rn input register
+ * \param width vector element width as immediate
```

Comment:
nit: All 4 of these are nicer as complete sentences: "The output register.", etc.

Maybe for the width you could reference OPND_CREATE_DOUBLE() + the other 2 (just trying to imagine what would make it easy for users).